### ISSUE TITLE
Language in banner 

### ISSUE THREAD

**Issue by jwinterm** [aeonix/aeonix.github.io#2]
Specifically these three lines:   
```
<h1><strong>AEON</strong> isn't a cryptocurrency.<br/><strong>It's a lifestyle.</strong></h1>
            <p class="lead"><strong>It's about polished perfection, attained by breaking the rules with calculated mastery of the art.<strong></p>
            <p class="lead">It's about respecting history and pushing innovation forward at the same time. It's about more than just math: it's a vision of a world where luxury is the same as entry-level, and the limits are the heavens themselves. <small>&copy; americanpegasus</small></p>
```  
There was some discussion on bitcointalk that this rhetoric is a bit over the top, and makes people think of timeshares or something, with even the quote's originator [suggesting it should be changed](https://bitcointalk.org/index.php?topic=1197508.msg17945817#msg17945817).  

I don't have anything in particular in mind, just opening an issue so people can put their thoughts down in a more focused thread for discussion.

**Comment by amoebatron** [comment]
I think we could probably afford to ditch the entire header as it's not really serving much purpose. Removing the dark background section and going straight to the white background bit below would certainly look cleaner too.

**Comment by keledoro** [comment]
I agree with the suggestion to ditch the entire header. Once we have crafted a nice sounding copy for the landing page, we could bring something like a header or landing section back, but for now ...

**Comment by iamsmooth** [comment]
Was #10 sufficient to resolve this or is more needed?

**Comment by sci-comp** [comment]
@aeon1234 That looks more like finished work than a preview. I would love to see added in!

**Comment by BigslimVdub** [comment]
Looks great but currently you can not run aeon core software on any mobile device. There are android versions of light wallets using mobile nodes. Yes the idea is for full mobile operation but Aeon is not there yet so having this information bannered at the top of the projects home page does not make sense currently imo.

**Comment by thriftyMinnow** [comment]
I like the suggested change.  Ditch the "lifestyle" part, and focus on lightweight, since we are a monero-lite.

**Comment by camthegeek** [comment]
I'm in agreement with @aeon1234 

His concept looks neat and doesn't make me cringe or look the other way. Perhaps move this from an ISSUE and submit changes as a PR?

**Comment by BigslimVdub** [comment]
I think we should integrate #52 now and close both of these. It’s been open a year now.

**Comment by aeon1234** [comment]
may be closed for all I care
